src/test/java/wooteco/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -14,23 +14,55 @@
 import org.springframework.http.MediaType;
 import wooteco.subway.dto.LineRequest;
 import wooteco.subway.dto.LineResponse;
+import wooteco.subway.dto.StationRequest;
 
 @DisplayName("지하철 노선 관련 기능")
 class LineAcceptanceTest extends AcceptanceTest {
-    LineRequest lineRequest1 = new LineRequest("1호선", "blue");
-    LineRequest lineRequest2 = new LineRequest("2호선", "green");
+    private final StationRequest gangnamStationRequest = new StationRequest("강남역");
+    private final StationRequest seolleungStationRequest = new StationRequest("선릉역");
+    private final LineRequest lineRequest1 = new LineRequest("1호선", "blue", 1L, 2L, 10);
```

Comment:
의미 있는 변수명을 써보시기 바랍니다.
테스트 코드니까 한글 변수명을 사용해보면 어떨까요

```suggestion
    private final LineRequest 지하철_1호선= new LineRequest("1호선", "blue", 1L, 2L, 10);
```